webapp/channels/src/components/logged_in_route/index.tsx

```diff
@@ -30,6 +31,7 @@ export default function LoggedInRoute(props: Props) {
                     {theme && (
                         <CompassThemeProvider theme={theme}>
                             <OnBoardingTaskList/>
+                            <CloudPreviewModal/>
```

Comment:
Could you break this up so that there's a separate component that one component that just checks the license/subscription and then lazily loads the rest of the modal? As-is, even though we're using `makeAsyncComponent` above, we're still loading all of the data and CSS for this modal regardless of what the user's license is.

You're already sort of doing that with the `CloudPreviewModal` and `PreviewModalController` components, so I think the change would be to move the `makeAsyncComponent` into the `CloudPreviewModal` where it imports the `PreviewModalController` and refactor the modal slightly to move importing the data into the controller component

Reply:
I've made some changes to address this. Lmk what you think.

Reply:
Than looks good to me. Thanks!